ote_sdk/ote_sdk/entities/scored_label.py

```diff
@@ -6,25 +6,55 @@
 
 import datetime
 import math
+from dataclasses import dataclass
+from typing import Optional
 
 from ote_sdk.entities.color import Color
 from ote_sdk.entities.id import ID
 from ote_sdk.entities.label import Domain, LabelEntity
 
 
+@dataclass
+class LabelSource:
+    """
+    This dataclass contains information about the source of a scored label.
+
+    For annotations, the id of the user who created the label and for predictions, the
+    id and model storage id of the model that created the prediction. When a user has
+    accepted a predictions as is, both the user id of the user who accepted and the
+    model/model storage id of the model that predicted should be filled in.
+
+    Note that usually the LabelSource is created with default values and the source is
+    set at a later stage.
```

Comment:
This comment doesn't fit here imho, the low-level SDK entities should be unaware of the application code that uses them.